docs/docs-content/clusters/edge/local-ui/cluster-management/local-to-central.md

```diff
@@ -0,0 +1,189 @@
+---
+sidebar_label: "Pair Local Cluster with Palette"
+title: "Pair Local Cluster with Palette"
+description:
+  "Learn how to move a local cluster created on hosts with airgap installation mode to be managed by a Palette instance."
+hide_table_of_contents: false
+sidebar_position: 55
+tags: ["edge"]
+---
+
+Clusters provisioned on hosts that are installed in the `airgap` installation mode are managed locally by Local UI. You
+have the option to transfer the management of a local cluster from Local UI to a Palette instance to be managed
+centrally by pairing the local cluster with a Palette instance.
+
+:::preview
+
+:::
+
+## Pair Local Cluster with Palette
+
+Moving a local cluster to central management is a two-step process. You first send a pairing request from your local
+cluster to Palette using Local UI. Then you must accept the pairing request and select matching profiles in Palettes to
+match the cluster stack.
+
+### Prerequisites
+
+- An active locally managed cluster. The cluster must be in the **Running** state and all nodes must be healthy.
+
+- An Edge host registration token. For more information on how to create registration tokens, refer to
+  [Edge Host Registration](../../site-deployment/site-installation/create-registration-token.md).
+
+- All Edge hosts in your cluster must be able to connect to the Palette instance you intend to use to manage the
+  cluster.
+
+- Your Palette environment has cluster profiles whose latest version exactly match all layers of the local cluster.
+  Refer to [Accept Pairing Request from Palette](#accept-pairing-request-from-palette) for details and examples.
+
+### Procedure
+
+#### Send Pairing Request from Local Cluster
+
+1. Log in to [Local UI](../host-management/access-console.md).
```

Comment:
```suggestion
1. Log in to  Local UI. For guidance on how to log into Local UI, check out the [Access Local UI](../host-management/access-console.md) guide.
```
I changed it, just so people don't think it's a URL like all other "Log in to Palette" statements.